styles/desk-style.css

```diff
@@ -0,0 +1,185 @@
+
+/* This applies from 600px onwards */
+@media (min-width: 992px) {
+  /* Header of the page */
+  .portfo-header {
+    margin-bottom: 40px;
+    margin-right: 50px;
+  }
+
+  .desk-nav {
+    display: flex;
+    justify-content: space-around;
+  }
+
+  #hamburger-btn {
+    display: none;
+  }
+
+  .desk-nav li {
+    padding-right: 20px;
+  }
+
+  .desk-nav li a {
+    color: #344563;
+  }
+
+  /* Start hero section */
+  .hero {
+    background-image: url("../images/header-bg-des.svg");
+    margin-bottom: 50px;
+  }
+
+  .hero-content {
+    margin-top: 5%;
+    margin-left: 18%;
+    margin-right: 15%;
+  }
+
+  .hero-content h1 {
+    font-size: 48px;
+  }
+
+  .hero-content p {
+    font-size: 20px;
+  }
+
+  .snapshoot {
+    display: none;
+  }
+
+  .snapshoot-div {
+    width: 544px;
+    height: 448px;
+  }
+
+  .desk-snapshoot {
+    width: 100%;
+    height: 100%;
+    padding-top: 30px;
+    display: block;
+  }
+
+  .work {
+    margin-left: auto;
+    margin-right: auto;
+    width: 1156px;
+  }
+
+  .work h1 {
+    font-size: 40px;
+    line-height: 52px;
+  }
+
+  .work-tonic {
+    flex-direction: row;
+    padding-right: 20px;
+  }
+
+  .reverse {
+    flex-direction: row-reverse;
+    margin-right: 50px;
+  }
+
+  .work-info {
+    padding: 30px;
+    width: 50%;
+  }
+
+  .tags {
+    width: 305px;
+    grid-template-columns: auto auto auto auto;
+  }
+
+  .tags > li {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 16px;
+    letter-spacing: 0.03em;
+  }
+
+  .tags .desk {
+    display: grid;
+  }
+
+  button {
+    margin: 44px 0;
+  }
+
+  .text-info {
+    margin-bottom: 40px;
+    margin-top: 40px;
+  }
+
+  .text-info ul {
+    justify-content: unset;
+    list-style: unset;
+    font-size: 18px;
+  }
+
+  .text-info ul .client-name {
+    color: #344563;
+    font-weight: 500;
+    line-height: 24px;
+  }
+
+  .text-info ul li {
+    padding-right: 30px;
+  }
+
+  .about-me {
+    flex-direction: row;
+    min-height: 700px;
+  }
```

Comment:
- [ ] And finally, in the about section columns are not aligned in one line and there is a typo. Please make both sides in one line (marked with pink line) and of course don't forget to change `langguages` to `Languages` (marked with lime line). That's it, please fix all the mentioned issues, **good luck 🍀**

#### Screenshot the marks
![image](https://user-images.githubusercontent.com/69011963/150569458-e17878d2-04f8-44d2-a59a-fc23433db343.png)